restconnector/detail/RequestHandler.cpp

```diff
@@ -47,35 +46,43 @@ void RequestHandler::listen_()
 
 void RequestHandler::operator() ( const server::request &request, server::response &response )
 {
-    server::string_type ip = source( request );
-    RestZeqTranslator restZeqTranslator;
+    const std::string& method = request.method;
 
-    if ( !restZeqTranslator.isCommandSupported( request.destination ) )
+    try
     {
-        response = server::response::stock_reply( server::response::bad_request,
-                                                  "Command not found." );
-    }
-    else
-    {
-        const std::string& method = request.method;
-        try
+        if( ( method == "PUT" ) || ( method == "POST" ) )
         {
-            const ::zeq::Event zeqEvent = restZeqTranslator.translate( request.body );
-
-            if( method == "PUT" )
-                processPUT_( zeqEvent );
-            else if( method == "GET" )
-                processGET_( zeqEvent );
-            else if( method == "POST" )
-                processPOST_( zeqEvent );
-
```

Comment:
PUT, GET, POST, DELETE and PATCH should be defined as constant variables (REST verbs)